backend/open_webui/models/files.py

```
@@ -197,8 +198,8 @@ def update_file_data_by_id(self, id: str, data: dict) -> Optional[FileModel]:
                 file.data = {**(file.data if file.data else {}), **data}
                 db.commit()
                 return FileModel.model_validate(file)
-            except Exception as e:
 
+            except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)